index.js

```diff
@@ -0,0 +1,32 @@
+let n = 16;
+const Container = document.getElementById("container");
+
+const Slider = document.getElementById("slider");
+const SliderInput = document.getElementById("sliderValue");
+Slider.oninput = function () {
+  SliderInput.textContent = this.value;
+  n = this.value;
+};
+
+const generateBtn = document.getElementById("generate");
+generateBtn.onclick = renderGrid;
+
+function renderGrid() {
+  let table = ``;
```

Comment:
There is a better way to do this manipulations called `fragment`, read about it here :https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment